Examples/Swift/ViewController.swift

```diff
@@ -19,15 +19,16 @@ enum ExampleMode {
 class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDelegate, VoiceControllerDelegate {
 
     // MARK: - IBOutlets
-    @IBOutlet weak var mapView: NavigationMapView!
```

Comment:
@bsudekum, it isn’t necessary to take the map view out of the storyboard in order to add and remove it from the view hierarchy. After all, an object can be managed by the storyboard without even being part of the view hierarchy (by putting it in the “dock” above a scene).

We should put the map view back in the storyboard, since we’re already using a storyboard to manage the rest of the example application’s interface. Make the outlet strong, so that the view controller shares ownership of the map view and the map view won’t go away as soon as we remove it from the view hierarchy.